icons/cooking-pot.svg

```diff
@@ -0,0 +1,17 @@
+
+<svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="24"
+    height="24"
+    viewBox="0 0 24 24"
+    fill="none"
+    stroke="currentColor"
+    stroke-width="2"
+    stroke-linecap="round"
+    stroke-linejoin="round"
+>
+    <line x1="2" y1="11" x2="22" y2="11"/>
+    <polyline points="10.3 5.42 9.82 3.48 13.7 2.51 14.18 4.45"/>
```

Comment:
There needs to be a whole that stays visible even when you have 3.9px stroke width in the handle, this is ugly but legal ![icon (47)](https://github.com/lucide-icons/lucide/assets/25524993/99ad55c4-4ae1-4fbd-8e5b-8b3f9102d3f6)

Reply:
Very nice icon btw, love me all food related icons that have a bit of swoosh.